styles/index.css

```diff
@@ -0,0 +1,25 @@
+.weather-header {
+    display: flex;
+    flex-direction: column;
+    grid-column: auto / span 2;
+    color: white;
+
+}
+body {
+    background-color: rgb(136, 195, 241);
+}
+
+  .orange {
+    color: rgb(255, 145, 49);
+  }
+#temperatureDisplay{
+  display: flex;
+  font-size: xx-large;
```

Comment:
Nice work with the styling, though I think you could have pushed yourself to add some CSS grid styling